### DISCUSSION TITLE
Set-PnPListItem Author / Editor with a user that does not exist in sharepoint online

### DISCUSSION THREAD

**Discussion by TanguyPa** [pnp/powershell#1684] [Q&A]
As part of a migration from sharepoint 2013 to sharepoint online, I would like to keep the Author/Editor information present in sharepoint 2013 for "history".
Some of these users do not exist in sharepoint online and azure active directory.
Is it possible to create or update a person column (Author/Editor or other column) using a user that does not exist.
When I try to do the Set-PnpListItem command, I get an error telling me that the user does not exist (which seems normal).
I tried the sharegate migration tool which does this, it keeps the user names in the columns (the users are not present in the Get-PnPUser list) but they are displayed in text format (you can't click on them to show information of user).
I want to reproduce this behavior with the pnp library, in order to be able to fill a column of type person with a string that acts as a username

**Comment by h-iobe** [comment]
I am also having the same problem.
Please let me know if there is a way to deal with it.
